src/meta-srv/src/metasrv.rs

```diff
@@ -52,6 +52,8 @@ impl Default for MetaSrvOptions {
     }
 }
 
+pub type Attrs = anymap::Map<dyn anymap::CloneAny + Send + Sync>;
```

Comment:
Seems using anymap is not a good idea, because you can't have same type with different meanings in it, which is a common case like a catalog_name String and a schema_name String.